### PR TITLE
Add Lane.depth and Lane.includes_language

### PR DESCRIPTION
**Pull request by leonardr**
This branch defines some utility methods for determining how deep a Lane is in a hierarchy and whether or not books in a given language might be found in that lane.

`Lane.depth` already existed but it was attached to the wrong class.

### CODE REVIEW

File: util/__init__.py [hunk 1]
```diff
@@ -580,9 +580,9 @@ def iso_639_2_for_locale(cls, locale):
             language, place = locale.lower().split("-",1)
         else:
             language = locale
-        if language in cls.two_to_three:
+        if cls.two_to_three[language]:
```
Comment (leonardr) [inline util/__init__.py]: I was encountering a problem where `'eng' in cls.two_to_three` was True even though `cls.two_to_three['eng']` was None. I haven't been able to reproduce this problem but this change is a more robust way to express the same underlying idea as the old code.

File: lane.py [hunk 2]
```diff
@@ -667,6 +667,22 @@ def set_list(field_name, value, default=None):
...
+    def includes_language(self, language):
+        """Would you expect to find books in the given language in
+        this lane?
+        """
+        if self.exclude_languages:
+            # We include all language except the ones on the exclude list.
+            if language in self.exclude_languages:
+                return False
+            else:
+                return True
```
Comment (daryanypl) [inline lane.py]: So, by including an exclude_languages, I could nullify any include_language specifications?

Comment (leonardr) [inline lane.py]: Yes, you can use _either_ languages or exclude_languages, not both.

Comment (daryanypl) [inline lane.py]: OK, was hoping that was intentional.

**Comment by daryanypl** [comment]
👍

**Pull request merged**